assets/prototype/domain/shared/predicates/dateUtilities/addSub.ts

```diff
@@ -0,0 +1,53 @@
+import {
```

Comment:
The fact that the file is named `dateUtilities/addSub.ts` won't matter that much because we will be using functions from `dateUtilities`, like:
`import { add } from '../../.. ../dateUtilities'`